powerlevel9k.zsh-theme

```diff
@@ -208,6 +208,14 @@ prompt_aws() {
   fi
 }
 
+# Current Elastic Beanstalk environment
+prompt_aws_eb_env() {
+  if [ -e .elasticbeanstalk/config.yml ]; then
+    local eb_env=$(cat .elasticbeanstalk/config.yml | grep environment 2> /dev/null | awk '{print $2}')
+    "$1_prompt_segment" "$0" black green "$(print_icon 'AWS_EB_ICON')  $eb_env"
```

Comment:
This should be now:
```zsh
"$1_prompt_segment" "$0" "$2" "black" "green" "$eb_env" 'AWS_EB_ICON'
```
Two parameters are new:
- Second parameter is now the `joined` flag to determine if a segment should be joined with preceding one.
- The last parameter is the visual identifier - a key of the `icons` array.